conf/battle/drops.conf

```diff
@@ -206,3 +206,7 @@ drop_rate_cap_vip: 9000
 
 // Displays a colored pillar effect for items dropped by monsters that contain random options.
 rndopt_drop_pillar: on
+
+// Drop rate bonus can increase drop rate from map drops.
+// Default: on
+enable_bonus_map_drops: on
```

Comment:
I wonder if we shouldn't stick to yes/no by default. Earlier I mistook "on" as "no".